apps/sms/views/shared/style/sms.css

```diff
@@ -261,3 +261,8 @@ html[dir="rtl"] .contact-header aside.pack-end {
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+/* Only change the opacity low storage disabled element */
+.low-storage-disabled {
```

Comment:
__nit:__ I think if we finally decide to do some changes via CSS in low storage condition maybe we can have smth like single ```low-storage``` css class applied to the body instead and use it to style particular elements?

__question:__ wondering if ```NewMessageView.beforeEnter``` can check low storage condition instead, show dialog and cancel navigation/disable controls (I assume we can do that if we return rejected promise in before enter)?

Or it doesn't work for all NewMessageView-related cases?

I also haven't look into the spec yet, but is it UX requirement to kind of disable Composer button instead of disabling controls in New Message View?